src/NzbDrone.Core/Indexers/TorrentPotato/TorrentPotato.cs

```diff
@@ -26,14 +26,6 @@ public TorrentPotato(IHttpClient httpClient, IIndexerStatusService indexerStatus
 
         }
 
-        public override IEnumerable<ProviderDefinition> DefaultDefinitions
-        {
-            get
-            {
-                yield return GetDefinition("Jackett", new TorrentPotatoSettings { BaseUrl = "http://localhost:9117/potato/YOURINDEXER"});
```

Comment:
With Jackett it's basicly the same as with TorrentPotato, just BaseUrl = "http://localhost:9117/torznab/YOURINDEXER" instead of BaseUrl = "http://localhost:9117/potato/YOURINDEXER"